docker/build_docker.sh

```diff
@@ -207,7 +207,23 @@ function docker_build {
         --label "org.opencontainers.image.authors=Demisto <containers@demisto.com>" \
         --label "org.opencontainers.image.version=${VERSION}" \
         --label "org.opencontainers.image.revision=${CIRCLE_SHA1}"
+
+    if [[ -e "dynamic_version.sh" ]]; then
+      echo "dynamic_version.sh file was found"
+      dynamic_version=$(docker run --rm -i "$image_full_name" sh < dynamic_version.sh)
+      echo "dynamic_version $dynamic_version"
+      VERSION="${dynamic_version}.${REVISION}"
+      image_full_name="${DOCKER_ORG}/${image_name}:${VERSION}"
+
+      # add the last layer and rebuild. Everything shuld be cached besides this layer
+      echo "ENV DOCKER_IMAGE=$image_full_name" >> "$tmp_dir/Dockerfile"
+      docker build -f "$tmp_dir/Dockerfile" . -t ${image_full_name} \
+        --label "org.opencontainers.image.authors=Demisto <containers@demisto.com>" \
+        --label "org.opencontainers.image.version=${VERSION}" \
+        --label "org.opencontainers.image.revision=${CIRCLE_SHA1}"
```

Comment:
Is the revision `${CIRCLE_SHA1}` or `${REVISION}`?

Reply:
${CIRCLE_SHA1} contains the SHA1 hash of the commit that triggered the current build in CircleCI. 
${REVISION} is the last 5 digits we have at the end of VERSION
im not sure why we call this label by this name, but this is how it was before also